### PR TITLE
Fix the label-on-approval workflow

### PR DESCRIPTION
**Pull request by necolas**
Resolves issues with https://github.com/facebook/lexical/pull/6120 (probably)

### CODE REVIEW

**Comment by Sahejkm** [comment]
- I don't think this works and auto triggers e2e tests, 1 hit workflow can't trigger another hit workflow as per the docs u less we use a pat token to run e2e tests

- 6120 is working fine for meta employees but not for external reviews becoz the GitHub actions don't have permissions for fork PRs and need pull request target event to work

File: .github/workflows/add-label-after-approved.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+on:
+  pull_request_review:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  label-after-approved:
+    if: github.event.review.state == 'approved' && !contains(github.event.pull_request.labels.*.name, 'extended-tests')
```
Comment (Sahejkm) [inline .github/workflows/add-label-after-approved.yml:12]: I tested this change locally by changing approved to commented, it doesn't auto trigger e2e tests unless some manual change on the PR

**Comment by necolas** [comment]
Got it. Updated to rename jobs for clarity (since it's less about labels and more about what happens after approval), and adopted the alternative trigger that you said would work for PRs from forks

**Review by Sahejkm — DISMISSED** [review]

**Comment by Sahejkm** [comment]
- I think it broke :( , I will test and create a pr for this as it was ald on my mind 😀

- we have to give both pull request and pull request target 

- we should specify branch also when using pull request target

**Comment by necolas** [comment]
Updated

**Review by necolas — COMMENTED** [review]
test review

**Review by Sahejkm — APPROVED** [review]
Thanks @necolas for enhancing this!

**Pull request merged**